WordPressKit/TransactionsServiceRemote.swift

```diff
@@ -35,44 +40,64 @@ import CocoaLumberjack
 
     public func createShoppingCart(siteID: Int,
                                    domainSuggestion: DomainSuggestion,
-                                   success: @escaping (String) -> Void,
+                                   privacyProtectionEnabled: Bool,
+                                   success: @escaping (CartResponse) -> Void,
```

Comment:
Seeing a Swift warning
`Expression implicitly coerced from 'Int?' to 'Any'`

Reply:
Not sure what this was about.  I ran a clean+rebuild and now I don't see the warning.  I call Xcode shenanigans.